src/services/jwt.service.ts

```diff
@@ -20,24 +21,27 @@ export class JWTService {
     });
   }
 
-  async decodeJWT(token: string):
-    Promise<string | { [key: string]: any } | null> {
+  async decodeJWT(
+    token: string
+  ): Promise<string | { [key: string]: any } | null> {
     return new Promise((resolve, reject) => {
       try {
         const decoded = jwt.decode(token);
         resolve(decoded);
       } catch (error) {
         reject(new Error('Error decoding JWT'));
       }
-    }
-    );
+    });
   }
 
-  async verifyJWT(token = ''): Promise<string | { [key: string]: any } | null> {
+  async verifyJWT(token = ''): Promise<AuthInterface.ITokenPayload> {
     return new Promise((resolve, reject) => {
       try {
-        const verify = jwt.verify(token, JWT_SECRET || '');
-        resolve(verify);
+        const decoded = jwt.verify(
+          token,
+          JWT_SECRET || ''
```

Comment:
Good point, but i think that define a `JWT_SECRET_DEFAULT` (example) in .env files makes no sense because in the sames files is already declared `JWT_SECRET` and with a problem loading this, there would be a problem loading the default too.
Did you thought about declare `JWT_SECRET_DEFAULT` in config file? (`src/config/index.ts`)

Reply:
yeap, defining a default there makes sense